src/dmifn.rs

```diff
@@ -589,15 +589,14 @@ pub fn dmi_processor_id(data: &SMBiosProcessorInformation<'_>) {
                     2 => {
```

Comment:
While we're at it, the comment // AMD should probably be verbose like the original C++ code to explain where this logic is coming from:
```cpp
// AMD, publication #25481 revision 2.28
```